libparsec/crates/platform_mountpoint/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "libparsec_platform_mountpoint"
+version = "0.0.0"
+edition = "2021"
+license = "BUSL-1.1"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+libparsec_types = { workspace = true }
+
+once_cell = { workspace = true }
+regex = { workspace = true }
+
+[dev-dependencies]
+libparsec_tests_fixtures = { workspace = true, features = ["default"] }
```

```suggestion
libparsec_tests_fixtures = { workspace = true }
```

using features = default means we depend on the other platform crates, which is not neeeded here